xmtp/src/conversation.rs

```diff
@@ -103,7 +103,7 @@ impl<'c, A> SecretConversation<'c, A>
 where
     A: XmtpApiClient,
 {
-    pub(crate) fn new(client: &'c Client<A>, peer_address: Address) -> Self {
+    pub fn new(client: &'c Client<A>, peer_address: Address) -> Self {
```

Comment:
Need to expose this to the bindings so that we can create Conversations on demand.